src/games/supported/Berzerk.cpp

```diff
@@ -130,3 +135,42 @@ void BerzerkSettings::loadState(Deserializer & ser) {
   m_lives = ser.getInt();
 }
 
+// returns a list of mode that the game can be played in.
+ModeVect BerzerkSettings::getAvailableModes(){
+    ModeVect modes(9);
+    for(unsigned i = 0; i < 9; i++){
```

Comment:
I changed the 16 17 18 to 0x10 0x11 0x12. Not sure the comment is still needed, but I put it in.